wad_project/socshare/views.py

```diff
@@ -8,15 +8,35 @@
 from socshare.utils.src import check_email
 from django.template.defaultfilters import slugify
 from django.db.models.functions.datetime import datetime
+from django.utils import timezone
 
 def events(request):
     search = request.GET.get('search')
-    events = Event.objects.filter(name__icontains=search) if search else Event.objects.order_by('date')
-    context = {"title":"Events","events":[x for x in events]}
+    eventsSearched = Event.objects.filter(name__icontains=search) if search else Event.objects.order_by('date')
+    context = {"title":"Events","events":[x for x in eventsSearched]}
```

Comment:
Unnecessary name change and doesn't follow PEP8 naming conventions

Reply:
Yep, That shouldve gone back to events, my bad